modules/ROOT/pages/release-notes/release-notes53.adoc

```diff
@@ -45,37 +46,38 @@ For example:
 |===
 | {productname} 5.2 toggle menu item checkmark behavior | {productname} 5.3 toggle menu item checkmark behavior
 
-| image:{baseurl}/images/old_togglemenuitemchecked.png[Custom toggle menu item with checkmark on the left-hand side of the item label (pre-5.3 behavior)]
-| image:{baseurl}/images/53_togglemenuitemchecked.png[Custom toggle menu item with checkmark on the right-hand side of the item label (5.3+ behavior)]
+| image:old_togglemenuitemchecked.png[Custom toggle menu item with checkmark on the left-hand side of the item label (pre-5.3 behavior)]
+| image:53_togglemenuitemchecked.png[Custom toggle menu item with checkmark on the right-hand side of the item label (5.3+ behavior)]
 |===
 
-For information on toggle menu items, see: link:{baseurl}/ui-components/menuitems/#togglemenuitems[Custom menu items - Toggle menu items].
+For information on toggle menu items, see: xref:ui-components/menuitems.adoc#togglemenuitems[Custom menu items - Toggle menu items].
```

Comment:
This xref doesn't resolve as the `togglemenuitems` id doesn't exist on the heading in `menuitems.adoc`. e.g:
```
[[togglemenuitems]]
=== Toggle menu items
```

Reply:
done